--- miniupnpc-1.9/Makefile.mingw.orig   2013-09-29 18:52:51.014087958 -1000
+++ miniupnpc-1.9/Makefile.mingw        2013-09-29 19:09:29.663318691 -1000
@@ -68,8 +68,8 @@
 
 wingenminiupnpcstrings.o:	wingenminiupnpcstrings.c
 
-miniupnpcstrings.h: miniupnpcstrings.h.in wingenminiupnpcstrings
-	wingenminiupnpcstrings $< $@
+miniupnpcstrings.h: miniupnpcstrings.h.in
+	sed -e 's|OS/version|MSWindows/5.1.2600|' -e 's|MINIUPNPC_VERSION_STRING \"version\"|MINIUPNPC_VERSION_STRING \"VERSIONHERE\"|' \$< > \$@
 
 minixml.o:	minixml.c minixml.h miniupnpcstrings.h
 
